src/global.css

```diff
@@ -91,3 +91,14 @@ body {
 [data-theme=dark] .react-flow__controls-interactive {
     border-radius: 0px 0px 8px 8px;
 }
+
+.clear-search-bar-button {
+    color: var(--chakra-colors-gray-300);
+    cursor: pointer;
+}
+.clear-search-bar-button:hover {
+    color: var(--chakra-colors-gray-000);
+}
+.clear-search-bar-button > * {
+    font-size: 66%;
+}
```

Comment:
How does this look in light theme?

Reply:
And why regular css instead of chakra props?

Reply:
Also the X should probably be darker in light mode, its kinda hard to see right now

Reply:
In that case, we should probably adjust the chakra variables because the magnifying glass isn't any better.

Reply:
The chakra variables you used are just colors. They don't change between dark and light. That's the reason I use useColorModeValue everywhere. If you really wanna do it in regular css, do it like how Zamboni did the controls UI where it selects light and dark theme and use the different color variables. But it's probably easier just to use the style props

Reply:
Done.

![image](https://user-images.githubusercontent.com/20878432/167868461-65a0b999-5eb8-417c-ab01-eda846543800.png)